src/builder/edit_channel.rs

```diff
@@ -115,4 +119,59 @@ impl EditChannel {
 
         self
     }
+
+    /// A set of overwrites defining what a user or a user carrying a certain role can
+    /// and cannot do.
+    ///
+    /// # Example
+    ///
+    /// Inheriting permissions from an exisiting channel:
+    ///
+    /// ```rust,no_run
+    /// # use serenity::{http::Http, model::id::GuildId};
+    /// # use std::sync::Arc;
+    /// #
+    /// # async fn run() -> Result<(), Box<dyn std::error::Error>> {
+    /// #     let http = Arc::new(Http::default());
+    /// #     let mut guild = GuildId(0).to_partial_guild(&http).await?;
+    /// use serenity::model::channel::{PermissionOverwrite, PermissionOverwriteType};
+    /// use serenity::model::id::UserId;
+    /// use serenity::model::permissions::Permissions;
+    ///
+    /// // Assuming a guild has already been bound.
+    /// let permissions = Some(PermissionOverwrite {
+    ///     allow: Permissions::READ_MESSAGES,
+    ///     deny: Permissions::SEND_TTS_MESSAGES,
+    ///     kind: PermissionOverwriteType::Member(UserId(1234)),
+    /// });
+    ///
+    /// guild.edit_channel(http, |c| {
```

Comment:
Guild models don't have an `edit_channel` method (though they should). Use [`GuildChannel::edit`](https://serenity-rs.github.io/serenity/current/serenity/model/channel/struct.GuildChannel.html#method.edit).